### ISSUE TITLE
Admonition Card and page path formatting issue

### ISSUE THREAD

**Issue by spestana** [rundocs/jekyll-rtd-theme#36]
**Describe the bug**
A change must have been made recently that's causing this.

Admonition Cards, and the page path at the top are no longer appearing properly (see https://mountain-hydrology-research-group.github.io/data-analysis/modules/week5.html)

appearance now:
![now](https://user-images.githubusercontent.com/650301/93687551-789d9e00-fa73-11ea-8e39-7076fa1d7b2b.JPG)

previous, correct appearance:
![previous](https://user-images.githubusercontent.com/650301/93687553-79363480-fa73-11ea-88db-5ec77803aa5b.JPG)

, and attached images)

**To Reproduce**

1. I had cached versions of github pages using the jekyll-rtd-theme still showing the correct previous behavior (second image above)
  * https://mountain-hydrology-research-group.github.io/data-analysis/modules/week5.html
  * https://rundocs.github.io/jekyll-rtd-theme/test/admonitions.html
2. Refreshing these pages so as to view a current version (first image above)

**Expected behavior**
Adminition cards and page paths should appear as they did previously

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Firefox 80.0.1 (64-bit)

**Comment by saowang** [comment]
Thank you for your report. This is an expected problem, because the 2.0 preview version has just been released, and you have just used it, and it is still in the testing phase. Now that the main branch has been rolled back to version 1.1.7, it can be rebuilt, thank you

**Comment by spestana** [comment]
Is there a way to use a specific version of the gem? My page doesn't seem to have reverted back to 1.1.7

I've tried specifying the version in my Gemfile like:
`gem 'jekyll-rtd-theme', '~> 1.1', '>= 1.1.7'`
This seems to have had no effect.

And in my _config.yml file like:
`remote_theme: rundocs/jekyll-rtd-theme@1.1.7`
This caused the page build to fail, so I removed it.

I'm not sure the proper way to force my page to use a specific version here.

Thanks for your help!
